app/templates/components/public/side-menu.hbs

```diff
@@ -3,32 +3,44 @@
     {{#scroll-to href='#info' class='item active'}}
       {{t 'Info'}}
     {{/scroll-to}}
-    {{#scroll-to href='#tickets' class='item'}}
-      {{t 'Tickets'}}
-    {{/scroll-to}}
-    {{#scroll-to href='#speakers' class='item'}}
-      {{t 'Speakers'}}
-    {{/scroll-to}}
+    {{#if event.tickets}}
```

Comment:
This will cause a network request to tickets. Use `is-ticketing-enabled` from events response for this.